### PR TITLE
Document Deploy and Play for process applications

### PR DESCRIPTION
**Pull request by mesellings**
## Description

Relates to and closes https://github.com/camunda/camunda-docs/issues/4182.

As per review comments on PR [OOTB PDLC Happy Path documentation](https://github.com/camunda/camunda-docs/pull/4144/files/2b740e7c55f10743250f3d6863e45eb8872ff29d#top), this PR update the "Deploy and Run process applications" page for 8.6.

Specifically:

* Between now and 8.6, it makes sense to deploy your process app since Play doesn't connect to a cluster.
* Starting with 8.6, you should use Play to run your process in a development cluster and only use the Deploy/Run modals for interacting with other stages.

## Acceptance criteria

- [x] Edit https://docs.camunda.io/docs/next/components/modeler/web-modeler/deploy-process-application/ to include Play as a heading/section, and specify when to use Play vs Run.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [X] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [X] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [X] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
@HanselIdes I've created a draft of the edits I think might be required for this work - please can you check to see what else we might need to mention here? I also wasn't sure about whether to rename the page, what do you think?

https://preview.docs.camunda.cloud/pr-4367/docs/next/components/modeler/web-modeler/deploy-process-application/

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 1]
```diff
@@ -8,13 +8,14 @@ import DeployImg from './img/process-applications/define-stages-deploy.png'
 import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
 import DeployErrorImg from './img/process-applications/deploy-error.png'
 
-You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+You can deploy your process application to a cluster, and play or run your process application to test, debug, and observe how it performs in a development or live environment.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/deploy-process-application.md:11]: This is starting to get really long, so let's see how it can be fixed. Here's the user journey:

A user is developing their process. They `play` their process to test, debug, and observe how it performs in a development environment.
After development, they `deploy` to a test, stage, or prod environment to run automated tests or have it available for a testing environment. For manual testing or as needed (it's more edge case-y), they can manually `run` their process.

I don't see a good use case for testing, debugging, or observing a process's performance in a live (production) environment.

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 2]
```diff
@@ -30,13 +31,32 @@ All BPMN, DMN, and form files contained in the process application folder are de
...
 :::
 
+## Play and validate your process application
+
+You can use [Play](/components/modeler/web-modeler/play-your-process.md) to run and validate your process application in a development cluster before deploying and running it in a testing, staging, or production cluster.
+
+To validate your process application:
+
+1. Open the [main process](create-a-process-application.md#main-process).
+1. Select the **Play** tab to start a Play environment using your development cluster.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/deploy-process-application.md:41]: The concept of a "play environment" was removed in 8.6. The play environment `is` the dev cluster

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 3]
```diff
@@ -30,13 +31,32 @@ All BPMN, DMN, and form files contained in the process application folder are de
...
+:::info
+To learn more about how to use Play for validation, see [play mode for rapid validation](/components/modeler/web-modeler/play-your-process.md)
+:::
+
 ## Run a process application
 
 You can run your process application to test, debug, and observe how it performs in a live environment.
 
+:::note
+You should use Play to run and validate your process application in a development cluster, and only use Run when interacting with other stages such as testing, staging, or production.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/deploy-process-application.md:53]: 💯

**Review by HanselIdes — CHANGES_REQUESTED** [review]

**Comment by mesellings** [comment]
Thanks for the feedback @HanselIdes - I've amended the page (including the title) to hopefully cover the things you have requested - let me know if more clarification is needed? (NB: Give the preview environment 20 mins to update)

**Review by HanselIdes — DISMISSED** [review]
This reorganization makes a lot of sense. Thank you @mesellings !

**Comment by mesellings** [comment]
@HanselIdes Please re-approve as I made a few small edits (meta description) and updates to the main page 👍

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 id: deploy-process-application
 title: Validate and deploy your process application
-description: You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+description: Validate your process application in development, before deploying it to testing, staging, or production.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/deploy-process-application.md:4]: Grammar?

**Review by HanselIdes — APPROVED** [review]
LGTM - one minor grammatical comment about comma placement

File: docs/components/modeler/web-modeler/deploy-process-application.md (hunk 4 above)
Comment (mesellings) [reply docs/components/modeler/web-modeler/deploy-process-application.md:4]: I went back and forth on this comma! I eventually kept it in as I wanted to make the user pause before the second concept (deployment) - without it, it felt like there was too much to take in in one go - does that make sense?

Comment (mesellings) [reply docs/components/modeler/web-modeler/deploy-process-application.md]: Changed in the end as suggested, perhaps having the comma doesn't work after all 🤷 - thank you!

**Review by HanselIdes — APPROVED** [review]
LGTM

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 5]
```diff
@@ -1,20 +1,33 @@
...
 import DeployImg from './img/process-applications/define-stages-deploy.png'
 import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
 import DeployErrorImg from './img/process-applications/deploy-error.png'
 
-You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+You should validate your process application in development before deploying it to testing, staging, or production.
+
+## Validate your process application
+
+Use [play mode](/components/modeler/web-modeler/play-your-process.md) to validate your process application in development.
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/deploy-process-application.md:15]: Do we want to capitalize Play consistently? If so, would update here.

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 6]
```diff
@@ -30,20 +43,23 @@ All BPMN, DMN, and form files contained in the process application folder are de
...
 :::
 
-## Run a process application
+## Run your process application
+
+You can also manually [run](/components/modeler/web-modeler/run-or-publish-your-process.md#run-a-process) your process application if required, for example to perform manual testing.
 
-You can run your process application to test, debug, and observe how it performs in a live environment.
+:::note
+You should use Play to validate your process application in a development cluster, and only use Run when interacting with other stages such as testing, staging, or production.
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/deploy-process-application.md:51]: Perhaps simply "Use Play to validate your process application in a development cluster, and only use Run when interacting with other stages such as testing, staging, or production." Or, add the reasoning behind this?

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 7]
```diff
@@ -1,20 +1,33 @@
...
+1. Perform validation as required, for example, debug your process logic and test the process application.
+
+:::info
+To learn more about using Play for validation, see [play mode for rapid validation](/components/modeler/web-modeler/play-your-process.md)
+:::
 
 ## Deploy your process application
 
-To deploy your process application:
+Deploy your process application to testing, staging, or production, for example to run automated tests or make it available for testing.
```
Comment (conceptualshark) [inline docs/components/modeler/web-modeler/deploy-process-application.md:27]: Linking back to the [deployment pipeline stages](https://docs.camunda.io/docs/next/components/modeler/web-modeler/process-application-pipeline/#deployment-pipeline-stages) might be useful here, and additional context:

"Once your application is validated, deploy it to the the cluster used in your [development lifecycle](https://docs.camunda.io/docs/next/components/modeler/web-modeler/process-application-pipeline/#deployment-pipeline-stages). The following steps are used to deploy your process application to any cluster environment, including production:"

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 8]
```diff
@@ -30,20 +43,23 @@ All BPMN, DMN, and form files contained in the process application folder are de
 If any resource fails to deploy, the whole deployment [fails](#deployment-errors) and the cluster state remains unchanged. This safely ensures that a process application cannot be deployed incompletely or in an inconsistent state.
 :::
 
-## Run a process application
+## Run your process application
+
+You can also manually [run](/components/modeler/web-modeler/run-or-publish-your-process.md#run-a-process) your process application if required, for example to perform manual testing.
```
Comment (conceptualshark) [inline docs/components/modeler/web-modeler/deploy-process-application.md:48]: Same sort of clarification/context here -

"To test a process once it has been deployed to a testing, staging, or production environment, use the **Run** option. Using Run once a process is deployed allows you to observe how your process performs in a live environment, and not only with test data provided using Play."

File: docs/components/modeler/web-modeler/process-applications.md [hunk 9]
```diff
@@ -33,11 +33,11 @@ Get started by creating a new process application.
 - A process application must always have a [main process](/docs/components/modeler/web-modeler/create-a-process-application.md#main-process). You can rename and reassign the main process at any time.
 - You can [add files](/docs/components/modeler/web-modeler/create-a-process-application.md#add-files-to-a-process-application) to the process application as required.
 
-## Deploy and run a process application
+## Validate and deploy your process application
 
-You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+You should validate your process application in development before deploying it to testing, staging, or production.
```
Comment (conceptualshark) [inline docs/components/modeler/web-modeler/process-applications.md:38]: If the overview section gets rewritten, this should be updated to match.

**Review by conceptualshark — COMMENTED** [review]
A little feedback on adding context/incorporating the user journey here, with some added explanations.

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 10]
```diff
@@ -1,20 +1,33 @@
...
+title: Validate and deploy your process application
+description: Validate your process application in development before deploying it to testing, staging, or production.
 ---
 
 import DeployImg from './img/process-applications/define-stages-deploy.png'
 import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
 import DeployErrorImg from './img/process-applications/deploy-error.png'
 
-You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+You should validate your process application in development before deploying it to testing, staging, or production.
```
[the remark this answers is not in the document]
Comment (mesellings) [reply docs/components/modeler/web-modeler/deploy-process-application.md:11]: Agreed with the first one - technically this is more of a strong recommendation as it assumes they follow the dev lifecycle we want them to (hence should rather than must), but I think your suggestion works - thank you 👍

File: docs/components/modeler/web-modeler/deploy-process-application.md (hunk 5 above)
Comment (mesellings) [reply docs/components/modeler/web-modeler/deploy-process-application.md:15]: Thanks, I thought we didn't capitalize links? Must be my misunderstanding - I'll re-capitalize 👍

[the remark this answers is not in the document]
Comment (mesellings) [reply docs/components/modeler/web-modeler/deploy-process-application.md:15]: Thanks, done 👍

**Review by HanselIdes — DISMISSED** [review]
LGTM (still)

File: docs/components/modeler/web-modeler/deploy-process-application.md (hunk 7 above)
Comment (mesellings) [reply docs/components/modeler/web-modeler/deploy-process-application.md:27]: Thanks, feedback incorporated 👍

File: docs/components/modeler/web-modeler/deploy-process-application.md (hunk 8 above)
Comment (mesellings) [reply docs/components/modeler/web-modeler/deploy-process-application.md:48]: Thanks, feedback incorporated 👍

File: docs/components/modeler/web-modeler/deploy-process-application.md [hunk 11]
```diff
@@ -1,20 +1,33 @@
...
+
+## Validate your process application
+
+Use [play mode](/components/modeler/web-modeler/play-your-process.md) to validate your process application in development.
+
+1. Open the [main process](create-a-process-application.md#main-process).
+1. Select the **Play** tab to play the process application using your selected development cluster.
```
[the remark this answers is not in the document]
Comment (mesellings) [reply docs/components/modeler/web-modeler/deploy-process-application.md:18]: Good spot, but I want to minimally document using Play here, that level of detail should remain in the Play docs - this is more of a brief overview of the process 👍

**Comment by mesellings** [comment]
@christinaausley @conceptualshark Please re-review for approval - great feedback - I have incorporated it all pretty much in one way or another, thank you! 👍 🚀

**Review by conceptualshark — APPROVED** [review]
👍 LGTM! Nice improvements to this section.

**Review by christinaausley — APPROVED** [review]
